build-system/tasks/ava.js

```diff
@@ -15,22 +15,34 @@
  */
 'use strict';
 
+const argv = require('minimist')(process.argv.slice(2));
 const {execOrDie} = require('../common/exec');
 
 /**
  * Runs ava tests.
  */
 async function ava() {
-  const avaCli = 'node_modules/ava/cli.js';
   const testFiles = [
     'build-system/tasks/get-zindex/get-zindex.test.js',
     'build-system/tasks/prepend-global/prepend-global.test.js',
-  ].join(' ');
-  execOrDie(`${avaCli} ${testFiles} --color --fail-fast`);
+  ];
+  execOrDie(
+    [
+      'npx ava',
+      ...testFiles,
+      '--color --fail-fast',
+      argv.watch ? '--watch' : '',
+    ].join(' ')
+  );
 }
 
 module.exports = {
   ava,
 };
 
 ava.description = "Runs ava tests for AMP's tasks";
+
+ava.flags = {
+  'watch':
+    '  https://github.com/avajs/ava/blob/main/docs/recipes/watch-mode.md',
```

Comment:
Copying comment from the other PR: Make this a human-readable sentence like `Watches for changes`, because it appears in the help documentation when someone runs gulp --tasks.